api/scripts/data-generation/add-many-fake-members-related-to-one-organization.js

```diff
@@ -0,0 +1,90 @@
+const { knex } = require('../../lib/infrastructure/bookshelf');
+const DomainTransaction = require('../../lib/infrastructure/DomainTransaction');
+const { MembershipUpdateError, UserCantBeCreatedError, ForbiddenAccess } = require('../../lib/domain/errors');
+const _ = require('lodash');
+
+const INITIAL_ID = 300000;
+
+function _buildMembership(iteration) {
+  const organizationId = process.argv[3];
+  const memberRole = process.argv[4];
+  const initialIdStart = parseInt(process.argv[5]) || INITIAL_ID;
+
+  return {
+    organizationRole: memberRole,
+    organizationId: organizationId,
+    userId: initialIdStart + iteration,
+    disabledAt: null,
+  };
+}
+
+function _buildUser(iteration) {
+  const initialIdStart = parseInt(process.argv[5]) || INITIAL_ID;
+  return {
+    firstName: `firstName${initialIdStart + iteration}`,
+    lastName: `lastName${initialIdStart + iteration}`,
+    email: `firstName.lastName-${initialIdStart + iteration}@example.net`.toLowerCase(),
```

Comment:
Question :
Si on met tout en dur et que seul les chiffres changent, pourquoi un toLowerCase ? 

```suggestion
    email: `firstname.lastname-${initialIdStart + iteration}@example.net`,
```

Reply:
Bien vu ! Je suis partie de l'example qu'il y a dans `api/db/database-builder/factory/build-user.js`, y'a eu des essais de boucle et... C'est resté ! Effectivement tout est en dur, y'a plus besoin, je corrige ^^